packages/core/src/init/initCore.ts

```
@@ -0,0 +1,8 @@
+import { NoteEditor } from "../notes";
+import { getBuiltinNotetypes } from "@/builtin/notetypes";
+
+export interface InitCoreOptions { }
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)